src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationClientFactory.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using Azure;
+using Azure.Data.AppConfiguration;
+using Azure.Core;
+using Microsoft.Extensions.Azure;
+using System.Collections.Generic;
```

Comment:
![image](https://github.com/user-attachments/assets/f3ec8cae-2894-42e2-9fe3-0d1fd0343a56)


It seems like there are unused packages. Could you verify it? BTW, could you also run `dotnet format` in the root folder to make sure import ordering follows alphabetical order。

Reply:
Thanks done